androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/Extensions.kt

```diff
@@ -20,3 +25,21 @@ fun Context.setPortrait() {
     val activity = this.findActivity()
     activity?.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
 }
+
+fun Modifier.fillMaxSizeOnLandscape(orientation: Int) =
+    when (orientation) {
+        Configuration.ORIENTATION_LANDSCAPE -> this.then(Modifier.fillMaxSize())
+        else -> this
+    }
+
+fun Context.toggleStatusBarVisibility(showStatusBar: Boolean) {
```

Comment:
nit: there is no toggle, because you are passing a boolean to define whether the status bar is visible or not

```suggestion
fun Context.setStatusBarVisibility(showStatusBar: Boolean) {
```